etcdserver/etcdserverpb/rpc.proto

```diff
@@ -469,6 +469,11 @@ message RoleRevokeRequest {
 
 message AuthEnableResponse {
   ResponseHeader header = 1;
+
+  enum Result {
```

Comment:
probably just remove the result thing if it does not fail. we can add result later. want to keep this minimal to start.